Library/Homebrew/requirements/cctools_requirement.rb

```diff
@@ -0,0 +1,8 @@
+class CctoolsRequirement < Requirement
+  fatal true
+  default_formula 'cctools'
+
+  satisfy do
+    MacOS::Xcode.installed? || MacOS::CLT.installed?
```

Comment:
I think it's probably good practice to add it in case this behaviour changes in future.